pkg/controller/core/queue_controller.go

```diff
@@ -20,24 +20,31 @@ import (
 	"context"
 
 	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog/v2"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
should be in the previous import block

Reply:
curious why the vscode/go extension didn't fixed this, it usually does. fixing